model/src/main/resources/reference.yml

```diff
@@ -152,7 +152,7 @@ ort:
         # Criteria for matching stored scan results. These can be configured for any scanner that uses semantic
         # versioning. Note that the 'maxVersion' is exclusive and not part of the range of accepted versions.
         minVersion: '3.2.1-rc2'
-        maxVersion: '30.2.0'
+        maxVersion: '32.0.0'
```

Comment:
Are you sure 32.0.0 will work without additional changes?

Reply:
No :grin: But `maxVersion` is *exclusive* anyway.

Reply:
Right, as the comment above says... Still unintuitive.